core/tests/protractor_utils/AdminPage.js

```diff
@@ -41,8 +42,9 @@ var AdminPage = function(){
           editingInstructions(forms.getEditor(objectType)(configProperty));
           saveAllConfigs.click();
           general.acceptAlert();
-          // Time is needed for the saving to complete.
-          browser.waitForAngular();
+          // Waiting for success message.
+          browser.wait(until.textToBePresentInElement(statusMessage,
+            'saved successfully'), 15000, 'New config could not be saved');
```

Comment:
15000 seems surprisingly long FYI (but my objection isn't a strong one).

Reply:
The editConfigProperty is not utilized anywhere in the suite. Are we concerned with testing e2e functionalities from an admin point of view?

Reply:
Let's keep it. I think it'll be needed later, and it would be good to have coverage for this.

Reply:
OK. Then I think leaving it at 15 seconds will be OK since there are a lot of configs on the page. Some can even be an entire HTML message so I was just pre-empting the wait time.

Reply:
Oh, they all get sent to the backend as a single AJAX request though. So I think you should be fine.

Anyway, I don't feel that strongly. Just thinking that when you want to abstract this stuff it would be better to not require configurable wait times.

Reply:
I was thinking about having a default 5000 ms unless a time variable is passed along, just like current waitForSystem. Would that be good?